index.html

```diff
@@ -174,35 +174,36 @@ <h3>Supply Visibility System</h3>
       <div class="container">
         <h2 class="section-title">initiatives</h2>
         <div class="columns">
-          <div class="column initiatives-image-column is-5">
-            <img class="initiatives-map-img" src="assets/initiatives-map.png" alt="utmap" />
+          <div class="column initiatives-image-column is-4">
+            <img class="initiatives-map-img" src="assets/initiatives-map.png" alt="utmap"/>
           </div>
           <div class="column initiatives-wrapper">
             <div class="initiatives-columns-wrapper">
               <div class="columns">
-                <div class="column content-column">
-                  <p class="column-initiatives">who are we and what we do</p>
-                <div class="column content2-column">
-                  <p class="column text">We are a team of architects, urbanists and software engineers who integrate their expertise to contribute to a development for a cooler future. </p>
-                </div>
-                <div class="column content2-column">
-                  <p class="column text">We have particular focus on grassroots level and are in search of programs who tackle vulnerable societies or troubled urban areas. We think the afore-mentioned tools and methodologies should be used in order to provide a better living for people around the globe.</p>
-                </div>
-                <div class="column content2-column">
-                  <p class="column  text">As the world we live in is changing at a fast pace, we thought of uniting our skills and wrapping them through a technological filter in order to adjust to the upcoming changes and tackle the everyday issues on a more expert level.</p>
-                </div>
-                <div class="column is-one-fifth"></div>
-                <div class="column">
-                  <ul>
-                    <li>+ Land value modelling</li>
-                    <li>+ Investement forecast</li>
-                    <li>+ Optimisation of public communication</li>
-                    <li>+ Green city corridors mapping</li>
-                    <li>+ City cycling grid optimisation</li>
-                    <li>+ Softwares and plugins conceptions</li>
-                    <li>+ Participation</li>
-                  </ul>
-                </div>
+                <div class="column initiatives">
```

Comment:
this line or the next one seem redundant. We have a nesting:

-- columns
---- column
------ column
------ column
------ column

whereas we should have
-- columns
---- column
---- column
---- column